src/qrCard/index.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+import { space } from '_utils/branding'
+
+import QrCard from './QrCard'
+
+const StyledQrCard = styled(QrCard)`
+  & {
+    display: inline-block;
```

Comment:
I would prefer to be a block by default.
The inline-block behavior is only specific to the future CardSection.